charts/netbox/templates/cronjob.yaml

```diff
@@ -122,6 +124,7 @@ spec:
           - name: config
             configMap:
               name: {{ include "common.names.fullname" . }}
+          {{- if (eq .Values.customSecretMount false) }}
```

Comment:
I don't think this is necessary.

```suggestion
```

Reply:
If the user mounts secrets directly onto the pods, these Kubernetes secrets won't exist. Making it necessary to have the option of disabling the creation of the volumes and volumeMounts mentioning these non-existent secrets.